packages/eslint-plugin/CHANGELOG.md

```diff
@@ -2,14 +2,15 @@
 
 ## Unreleased
 
+### New Features
+
+-   Enabled `import/default` and `import/named` rules in the `recommended` ruleset. [#28513](https://github.com/WordPress/gutenberg/pull/28513)
+
 ## 8.0.1 (2021-01-28)
 
 ### Bug Fix
 
 -   Add missing `eslint-plugin-import` npm dependency ([#28545](https://github.com/WordPress/gutenberg/pull/28545)).
-
-## 8.0.0 (2021-01-21)
-
```

Comment:
It looks like a merge issue. It should not get removed.

Reply:
Oops! Thanks!